build_tools/github_actions/lint_clang_format.sh

```diff
@@ -51,7 +49,12 @@ fi
 echo "Running clang-format [mode=$FORMAT_MODE]..."
 echo "  Files: " "${CHANGED_FILES[@]}"
 if [[ $FORMAT_MODE == 'fix' ]]; then
-  clang-format --style=google -i "${CHANGED_FILES[@]}"
+  clang-format -i "${CHANGED_FILES[@]}"
```

Comment:
why get rid of the `--style`? I'm forgetting why it was added but I think there were issues when removing this.. can you try running this clang format command on the repo and see how much changes?

Reply:
We have a `.clang-format` file which specifies `BasedOnStyle: Google`, so the flag is not needed.